FWCore/Framework/src/StreamSchedule.cc

```diff
@@ -81,51 +81,48 @@ namespace edm {
       return ptr;
     }
 
-    void initializeBranchToReadingWorker(ParameterSet const& opts,
+    void initializeBranchToReadingWorker(std::vector<std::string> vBranchesToDeleteEarly,
                                          ProductRegistry const& preg,
                                          std::multimap<std::string, Worker*>& branchToReadingWorker) {
-      // See if any data has been marked to be deleted early (removing any duplicates)
-      auto vBranchesToDeleteEarly = opts.getUntrackedParameter<std::vector<std::string>>("canDeleteEarly");
-      if (not vBranchesToDeleteEarly.empty()) {
```

Comment:
Decided not to keep the optimization?

Reply:
This is now handled in `EventProcessor` by calling the `initializeEarlyDelete()` (that ends up calling this function) only if the `canDeleteEarly` is not empty
https://github.com/cms-sw/cmssw/blob/70829468cc89bee72d6c5205ed931ca1609bb2bd/FWCore/Framework/src/EventProcessor.cc#L649-L650